.github/workflows/tests.yaml

```diff
@@ -0,0 +1,236 @@
+name: NATS Server Tests
+
+on:
+  # push:
+  # pull_request:
+  schedule:
+    - cron: "0 * * * *" # Hourly
```

Comment:
Might want to avoid 0 simply to avoid everyone else also using 0 and having to wait a long time for resources to become available.  Getting lost in the stampede ...

Reply:
Good point, set to `15` instead.